src/components/Calculator/Calculator.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import './Calculator.css';
+
+class Calculator extends Component {
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/164143217-b9d0411e-8b84-44e9-b1fe-fe6e30778ae3.png)

Your styling matches the shape mostly, but not the colors:

![image](https://user-images.githubusercontent.com/57421823/164143378-bb9c74ac-923f-4942-86ab-c886a30fecb1.png)


Therefore:

- [ ] Please, make your calculator match the styling exactly as in the given design, including the colors.

That way, your calculator will look beautiful 💪